readme_cn.md

```diff
@@ -0,0 +1,194 @@
+```
+
+### 显示图标
+
+想要显示图标, 你可以传递 `show_icons=true`，像这样:
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true)
+```
+
+### 自定义
+
+你可以为你的 `Stats Card` 或 `Repo Card` 自定义样式，however you want with URL params.
+
+自定义选项:
+
+| Option      | type      | description                          | Stats Card (default) | Repo Card (default) |
+| ----------- | --------- | ------------------------------------ | -------------------- | ------------------- |
+| title_color | hex color | title color                          | 2f80ed               | 2f80ed              |
+| text_color  | hex color | body color                           | 333                  | 333                 |
+| icon_color  | hex color | icon color                           | 4c71f2               | 586069              |
+| bg_color    | hex color | card bg color                        | FFFEFE               | FFFEFE              |
+| line_height | number    | control the line-height between text | 30                   | N/A                 |
+| hide_rank   | boolean   | hides the ranking                    | false                | N/A                 |
+| hide_title  | boolean   | hides the stats title                | false                | N/A                 |
+| hide_border | boolean   | hides the stats card border          | false                | N/A                 |
+| show_owner  | boolean   | shows owner name in repo card        | N/A                  | false               |
+| show_icons  | boolean   | shows icons                          | false                | N/A                 |
+
+- 你也可以通过自定义来兼容暗黑模式
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&title_color=fff&icon_color=79ff97&text_color=9f9f9f&bg_color=151515)
+```
+
+### Demo
+
+- 默认
+
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)
+
+- 隐藏某个项目
+
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=["contribs","issues"])
+
+- 显示图标
+
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=["issues"]&show_icons=true)
+
+- 自定义添加卡片
+
+![Anurag's github stats](https://github-readme-stats.vercel.app/api/?username=anuraghazra&show_icons=true&title_color=fff&icon_color=79ff97&text_color=9f9f9f&bg_color=151515)
+
+- 自定义仓库卡片
+
+![Customized Card](https://github-readme-stats.vercel.app/api/pin?username=anuraghazra&repo=github-readme-stats&title_color=fff&icon_color=f9f9f9&text_color=9f9f9f&bg_color=151515)
+
+# GitHub Extra Pins
+
+GitHub extra pins allow you to pin more than 6 repositories in your profile using a GitHub readme profile.
+
+Yey! You are no longer limited to 6 pinned repositories.
+
+### Usage
+
+Copy-paste this code into your readme and change the links.
+
+Endpoint: `api/pin?username=anuraghazra&repo=github-readme-stats`
+
```

Comment:
LGTM, You missed this part i think.

Reply:
So sorry, I forgot to explain for it.

I have some puzzle in this part, maybe I know the meaning, but I can't translate it exactly.
So I choose not to translate it.

If you think it's necessary to do that, I will try it. But I think that keep as it is is the better.

Reply:
I see, no problem then, LGTM! Thanks!